stable/grafana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: grafana
-version: 1.6.0
+version: 1.6.1
```

Comment:
Minor bump, please: 1.7.0

Reply:
Sorry, too early in the morning - fixed now :)